dataland-backend-utils/src/main/kotlin/org/dataland/datalandbackendutils/utils/EncodingUtils.kt

```diff
@@ -0,0 +1,38 @@
+package org.dataland.datalandbackendutils.utils
+
+import java.util.*
```

Comment:
Import mit Stern sind nicht zulässig

Reply:
Ich dachte eigentlich das detekt eine solche Regel hat, aber scheinbar aktuell nicht. Haben wir das geändert?

Reply:
Ah java.util ist by default als Ausnahme von dieser Regel gesetzt

Reply:
Aber dann sollte das "import java.util.zip.CRC32" hierunter eigentlich wegfallen dürfen oder?
Wird ja dann durch die Wildcard abgedeckt. Oder nicht?

Reply:
Nein der Stern bezieht sich nur auf Dinge die direkt in utils sind, es importiert aber nicht alle subpackages. Kannst du einfach ausprobieren durch auskommentieren der Zeile. Die IDE meckert dann.